flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -644,10 +644,16 @@ extractExecutionParams() {
     local execution_config=$1
     local EXECUTION_PREFIX="BASH_JAVA_UTILS_EXEC_RESULT:"
 
-    if ! [[ $execution_config =~ ^${EXECUTION_PREFIX}.* ]]; then
+    local num_lines=$(echo "$execution_config" | wc -l)
+    if ! [[ ${num_lines} == 1 ]]; then
+        echo "[ERROR] Unexpected result ($num_lines lines): $execution_config" 1>&2
+        echo "[ERROR] extractExecutionParams only accepts exactly one line as the input" 1>&2
+        exit 1
+    fi
```

Comment:
I think this change broke my local setup. If I build from master I can no longer start a cluster via `bin/start-cluster.sh`.

Reply:
I think the problem is that we don't do proper trimming.